jujugui/static/gui/src/app/components/added-services-list/added-services-list.js

```diff
@@ -37,7 +36,9 @@ class AddedServicesList extends React.Component {
         lastInList={lastInList}
         ref={'AddedServicesListItem-' + service.get('id')}
         service={service}
-        serviceModule={this.props.serviceModule} />);
+        serviceModule={this.props.serviceModule}
+      />
+    );
```

Comment:
I find this much easier to read and understand.

Indentation is a pointer to the structure of the code, having things open at different indentations and close all on one confuses my simple mind :face_with_head_bandage: